### PR TITLE
Migrating VSTS Account creation support from PE to ARM

### PR DESCRIPTION
**Pull request by kuvinodms**
Migrating VSTS Account creation support from PE to ARM

### CODE REVIEW

File: vsts_cd_manager/continuous_delivery_manager.py [hunk 1]
```diff
@@ -106,18 +119,13 @@ def setup_continuous_delivery(self, azure_deployment_slot, app_type, vsts_accoun
...
-            account_created = not creation_results.account_id == None
-            if account_created:
-                self._update_progress(5, 100, 'Team Services account created')
+            file_path = os.path.join(os.path.dirname(__file__), 'template.json')
+            deployment_name = 'VSTS.Account.Creation'
+            deployment_mode='incremental'
+            no_wait=False
+            parameters = self.get_parameters(vsts_account_name)
+            deploy_arm_template(self._azure_info.resource_group_name, file_path, None, deployment_name, parameters, deployment_mode, no_wait)
+            self._update_progress(5, 100, 'Team Services account created')
```
Comment (jpricket) [inline vsts_cd_manager/continuous_delivery_manager.py:128]: This code path should also work if the account already exists. Make sure to test that case. If you can detect that case, you may want to change the progress message to say created only if it was actually created.

File: vsts_cd_manager/continuous_delivery_manager.py [hunk 2]
```diff
@@ -106,18 +119,13 @@ def setup_continuous_delivery(self, azure_deployment_slot, app_type, vsts_accoun
         account_created = False
         accountClient = Account('3.2-preview', None, self._azure_info.credentials)
         if create_account:
-            # Try to create the account (already existing accounts are fine too)
```
Comment (jpricket) [inline vsts_cd_manager/continuous_delivery_manager.py:109]: Make sure the run the tests and consider adding some unit tests to cover this new code.

**Review by jpricket — APPROVED** [review]
The changes seem reasonable, but there may be some minor behavior changes (see comments)

**Comment by kuvinodms** [comment]
Hi Jason,
We've decided to migrate to AEX instead of ARM. So this code change need to scrap out. Hence closing the PR.
Thanks!

**Pull request closed without merging**